DirectProgramming/DPC++FPGA/Tutorials/Features/read_only_cache/README.md

```diff
@@ -0,0 +1,261 @@
+# Read-Only Cache 
+This FPGA tutorial demonstrates how to use the read-only cache feature to boost
+the throughput of an FPGA DPC++ design that uses a read-only look-up table
+defined in host code.
```

Comment:
That's a very specific use case.  Is this tutorial not more broadly applicable?  In other words, are there other cases where the read-only cache feature that this tutorial demonstrates would be useful?

Reply:
Addressed this by specified the general use case where the cache can be useful.